code/modules/crafting/recipes.dm

```diff
@@ -910,6 +910,16 @@
 	pathtools = list(/obj/item/reagent_containers/glass/rag = 1) //need something to paint with it
 	category = CAT_MISC
 
+/datum/crafting_recipe/tactical_medkit
+	name = "Tactical First-Aid Kit"
+	result = list(/obj/item/storage/firstaid/tactical/empty)
```

Comment:
This will create a medbot that only heals syndicate agents. You may wish to make a fake subtype that doesn't only heal agents, and doesnt inject perf / bicaridine / kelotine

Reply:
Oh! That's good to know. I'll change that.